### PR TITLE
Make blend files use the Blender icon

### PR DESCRIPTION
**Pull request by SamuelBilek**
Many addons use their own blend files. This change makes the blend files in VS Code display the Blender icon. The icon was taken from the official Blender repo.

### CODE REVIEW

**Comment by SamuelBilek** [comment]
@Mateusz-Grzelinski thanks for pointing it out! I found out that changing the file icons this way changes icons for all files so if they do not have defined icons they will be empty. I changed the `package.json` to change the blend icons in another way that does not do this. The only questionable thing here is that we define `blend` as a language but IMO it's ok this way. What do you think?

**Comment by Mateusz-Grzelinski** [comment]
Defining blend as language is ok as long as it does not break other blender extensions in va code store, but I do not know how to check it...
I assume that defining language does not do anything to provide functionality like " open file with blender"?

**Comment by SamuelBilek** [comment]
From the [API docs](https://code.visualstudio.com/api/language-extensions/language-configuration-guide) it seems to be used to define configurations for formatting files of a language. We could technically override definitions from other extensions and we cannot be sure if another extension defines them, but should we be concerned about editing blend files through VS Code?

I added the icons also for `blender.exe` and `blender-launcher.exe` :)

Also I think the idea of being able to open the blend files with Blender executables from this extension sounds really cool and I would like to implement it :)

File: package.json [hunk 1]
```diff
@@ -261,6 +261,18 @@
...
+    "languages": [
+      {
+        "id": "blend",
+        "extensions": [".blend"],
+        "aliases": ["Blend File", "blend"],
+        "icon": {
+          "light": "./icons/blender_icon.ico",
+          "dark": "./icons/blender_icon.ico"
+        },
+        "filenames": ["blender.exe", "blender-launcher.exe"]
```
Comment (Mateusz-Grzelinski) [inline package.json:274]: Linux and Mac do not have exe extensions 😋
And I guess I will not torture you with #160

Comment (SamuelBilek) [reply package.json]: Added variants without extensions 😺

File: package.json [hunk 2]
```diff
@@ -261,6 +261,18 @@
           }
         }
       }
+    ],
+    "languages": [
+      {
+        "id": "blend",
+        "extensions": [".blend"],
```
Comment (Mateusz-Grzelinski) [inline package.json:268]: Oh, btw, does automatic save in blender still adds number at the end of file creating new file extensions?
Like .blend1, .blend2... I think it used to do that back in a day

Comment (SamuelBilek) [reply package.json:268]: AFAIK this still works, but IMO displaying icons for all the `.blend*` backup files will clutter the UI. I think we should just display the Blender icon for main `.blend` files and keep the backups 'grayed out'.

**Comment by Mateusz-Grzelinski** [comment]
tested again with newest master, looks good.

**Pull request merged**